### ISSUE TITLE
Is there a plan for merging hs._asm.axuielement to hammerspoon?

### ISSUE THREAD

**Issue by zgqq** [Hammerspoon/hammerspoon#1975]
It is a very powerful module, unfortunately, still not a official module.

**Comment by latenitefilms** [comment]
Yes, I believe it's definitely on @asmagill's to-do list, but no ETA.

We've been using it in CommandPost for a very long time now, and it's rock solid. We'd also love to see it built-in to Hammerspoon natively, but in the meantime, there's really no limitation or down-side to having it as a 3rd party extension.

Related: #1630

**Comment by latenitefilms** [comment]
@cmsj - This can probably be closed in favour of #1630.
